candies/cli/args/opt.py

```diff
@@ -25,3 +30,6 @@ def say(phrase: Arg, times: Opt):
     def __init__(self, description: str = None, short: str = None):
         self.description = description
         self.short = short
+
+    def __class_getitem__(cls, type: Type) -> Typed:
```

Comment:
Not sure, but is such `__class_getitem__` implementation a good candidate for a metaclass like `MetaArg`?

Reply:
I don't know, to be honest. I think that this is an unnecessary complication for two very simple `__class__getitem__` methods.

Reply:
Yeah, I just never had an opportunity to introduce a meta-class in my life. :D

Reply:
I agree, such an opportunity is tempting. ))